### DISCUSSION TITLE
What is the purpose of the PDB files that get shipped with the installer?

### DISCUSSION THREAD

**Discussion by bastik-1001** [sandboxie-plus/Sandboxie#3748] [Q&A Feedback] [answered]
Do the PDB files that are shipped with the Sandboxie-Plus installer, serve a purpose relevant to the end-user, or are they becoming only relevant if someone is going to debug a component for which such a file is shipped?

**Comment by DavidXanatos** [comment, marked as answer]
They are used for debugging, and help with trace logs, 
SbieDrv.pdb is for the driver and used to investigate BSOD's
SbieDll.pdb is usefull when the stack trace featue in the trace log of sandman UI is enabled, then for every log event you have a full stack trace and see which function in SbieDll.dll caused the log.
